### PR TITLE
Fixes #15578: Add rudder-api-client to the rudder packages

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15578

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.DEFAULT_GOAL := build
+
+RUDDER_VERSION_TO_PACKAGE =
+RUDDER_MAJOR_VERSION := $(shell echo ${RUDDER_VERSION_TO_PACKAGE} | cut -d'.' -f 1-2)
+PYTHON = python3
+
+DESTDIR = $(CURDIR)/target
+
+build:
+	mkdir -p $(DESTDIR)
```
Comment (peckpeck) [inline Makefile]: This should be in install not in build

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+.DEFAULT_GOAL := build
+
+RUDDER_VERSION_TO_PACKAGE =
+RUDDER_MAJOR_VERSION := $(shell echo ${RUDDER_VERSION_TO_PACKAGE} | cut -d'.' -f 1-2)
+PYTHON = python3
```
Comment (peckpeck) [inline Makefile]: where is this used ?

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**